articles/connector/test-dc.md

```diff
@@ -76,48 +70,46 @@ You can run your VM on any cloud platform, but this guide will walk through how
+1. Open **Internet Explorer** with the **Ticket URL** you saved in step 1.
+1. Follow the instructions in the browser to download, install, and configure the **Connector**. When you are prompted for the LDAP service account, use the admin account you created for the VM:
+    * Username: `mycompany\ad-admin`
+    * Password: (same as before)
+1. When you're done configuring and installing the Connector, reboot the server.
+1. Log back into the VM using Remote Desktop.
+1. Open the Connector configuration site by navigating to `http://localhost:8357/`.
+1. Check that the **Connector** is able to find a user:
+    * Click on the **Search** tab.
+    * Under "Find User by Login", type `mary.smith`.
+    * Click **Search**. You should get JSON back that contains that user's AD profile data:
+    ![](/media/articles/connector/test-dc/test-find-user.png)
 
 ## Test an Authentication Flow from Auth0
 
 To ensure that everything is working using your Auth0 account, we're going to configure your **Default App** in Auth0 to use your new **Active Directory / LDAP** Connection, and use the `/authorize` endpoint to initiate an authentication flow.
 
 1. Using the [Auth0 Management Dashboard](${manage_url}), navigate to your [Clients](${manage_url}/#/clients).
-2. Click the **Settings** icon of your **Default App**
-3. Add `http://jwt.io` to the list of the Client's **Allowed Callback URLs**.
-4. Click the **Connections** tab.
-5. Under **Enterprise**, enable the `auth0-test-ad` **Active Directory / LDAP** connection.
-6. Test the authentication flow by opening the following link in your browser:  
-```
-https://${account.namespace}/authorize?response_type=token&scope=openid%20profile&client_id=${account.clientId}&redirect_uri=http://jwt.io&connection=auth0-test-ad
-```
-
-7. Log in with one of the test users that was created in the directory:
-  * Username: `mary.smith` or `bob.johnson`
-  * Password: `Pass@word1!`
-  ![](/media/articles/connector/test-dc/auth-flow-login.png)
-8. If everything is working, you should be redirected to the JWT.io website to see the contents of the resulting JWT:  
-  ![](/media/articles/connector/test-dc/auth-success.png)
+1. Click the **Settings** icon of your **Default App**
+1. Add `http://jwt.io` to the list of the Client's **Allowed Callback URLs**.
+1. Click the **Connections** tab.
+1. Under **Enterprise**, enable the `auth0-test-ad` **Active Directory / LDAP** connection.
+1. Test the authentication flow by opening the following link in your browser:
+    ```html
+    https://${account.namespace}/authorize?response_type=token&scope=openid%20profile&client_id=${account.clientId}&redirect_uri=http://jwt.io&connection=auth0-test-ad
+    ```
+
+1. Log in with one of the test users that was created in the directory:
+    * Username: `mary.smith` or `bob.johnson`
+    * Password: `Pass@word1!`
+    ![](/media/articles/connector/test-dc/auth-flow-login.png)
```

Comment:
This looks like that:

![image](https://user-images.githubusercontent.com/11715799/29020126-a6c29f74-7b69-11e7-957e-67debd4e24ae.png)